tests/test_core.py

```diff
@@ -1859,6 +1859,12 @@ def test_memorygrowth_3_force_fail_reallocBuffer(self):
     self.emcc_args += ['-s', 'ALLOW_MEMORY_GROWTH=1', '-s', 'TEST_MEMORY_GROWTH_FAILS=1']
     self.do_run_in_out_file_test('tests', 'core', 'test_memorygrowth_3')
 
+  @no_asmjs()
+  def test_module_wasm_memory(self, js_engines=None):
+    self.emcc_args += ['--pre-js', path_from_root('tests', 'core', 'test_module_wasm_memory.js')]
+    src = open(path_from_root('tests', 'core', 'test_module_wasm_memory.c')).read()
+    self.do_run(src, 'success', force_c=True, js_engines=js_engines)
```

Comment:
I think `, js_engines=js_engines` is not necessary.

I also think you can use `self.do_run_in_out_file_test('tests', 'core', 'test_module_wasm_memory')` for all these 3 lines.

Reply:
This doesn't seem possible since that doesn't allow providing a pre-js file and also requires the expected output to be in a separate file, and I don't think its a good idea to have a file with only a single word: "success".